BlocSpot/app/src/main/java/com/izor066/android/blocspot/api/model/database/table/PointsOfInterestTable.java

```diff
@@ -0,0 +1,91 @@
+package com.izor066.android.blocspot.api.model.database.table;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+/**
+ * Created by igor on 22/9/15.
+ */
+public class PointsOfInterestTable extends Table {
+
+    public static class Builder implements Table.Builder {
+
+        ContentValues values = new ContentValues();
+
+        public Builder setTitle(String title) {
+            values.put(COLUMN_TITLE, title);
+            return this;
+        }
+
+        public Builder setAddress(String address) {
+            values.put(COLUMN_ADDRESS, address);
+            return this;
+        }
+
+        public Builder setLongitude(float longitude) {
+            values.put(COLUMN_LONGITUDE, longitude);
+            return this;
+        }
+
+        public Builder setLatitude(float latitude) {
+            values.put(COLUMN_LATITUDE, latitude);
+            return this;
+        }
+
+        @Override
+        public long insert(SQLiteDatabase writableDB) {
+            return writableDB.insert(PointsOfInterestTable.NAME, null, values);
+        }
+    }
+
+    public static String getTitle(Cursor cursor) {
+        return getString(cursor, COLUMN_TITLE);
+    }
+
+    public static String getAddress(Cursor cursor) {
+        return getString(cursor, COLUMN_ADDRESS);
+    }
+
+
+
+    public static float getLatitude(Cursor cursor) {
+        return getFloat(cursor, COLUMN_LATITUDE);
+    }
+
+    public static float getLongitude(Cursor cursor) {
+        return getFloat(cursor, COLUMN_LONGITUDE);
+    }
+
+    public static Cursor getRowTest(SQLiteDatabase readonlyDatabase, long rowId) {
+        return readonlyDatabase.query(true, NAME, null, COLUMN_ID + " = ?",
+                new String[]{String.valueOf(rowId)}, null, null, null, null);
+    }
+
+    public static Cursor getRowFromTitle(SQLiteDatabase readonlyDatabase, String title) {
+        return readonlyDatabase.query(true, NAME, null, COLUMN_TITLE + " = ?",
+                new String[]{title}, null, null, null, null);
+    }
+
+    private static final String NAME ="points_of_interest";
```

Comment:
As I mentioned before, these fields (which are in fact constants here), should be moved at the top of the class.